troubleshooting/known_issues/missing_binaries_in_aarch64_images.md

```diff
@@ -0,0 +1,120 @@
+# Missing binaries in aarch64 Images
+
+Due to a bug in the QEMU emulation software, there are times that dependencies are missed
+for packages that are being installed on `aarch64` images when run in emulation on `x86_64`
+hardware. This will usually manifest when the image is being booted or running processes
+where an error about missing shared libraries is encountered.
+
+## Root Cause
```

Comment:
Other than the main title heading, we use sentence casing (capitalized like you would a regular sentence)

```suggestion
## Root cause
```